packages/core2/src/managers/_PluginsManager.ts

```diff
@@ -16,4 +20,10 @@ export class PluginsManager extends BaseManager {
 
 		await this.sliceMachinePluginRunner.init();
 	}
+
+	dangerouslyCallHook: SliceMachinePluginRunner["callHook"] = (...args) => {
+		assertPluginsInitialized(this.sliceMachinePluginRunner);
+
+		return this.sliceMachinePluginRunner.callHook(...args);
+	};
```

Comment:
I needed it in the init command to call the `command:init` hook, I didn't see it fit anywhere else 🤔

Reply:
Hmm, a dedicated `triggerCommandInitHook()` or something like it would be weird.

We could get the plugin runner itself and make the call outside the managers:

```ts
const pluginRunner = manager.getSliceMachinePluginRunner();

await pluginRunner.callHook("command:init");
```

This signals explicitly working outside the manager's API and dropping down to the lower-level plugin runner.

---

Do types work properly when using `dangerouslyCallHook()`? I thought it required a generic for it to infer the hook type and its arguments.